shell.nix

```diff
@@ -1,34 +1,27 @@
 { jdk ? "jdk17" }:
 
 let
-  config = {
-    packageOverrides = p: rec {
-      java = p.${jdk};
-
-      sbt = p.sbt.overrideAttrs (
-        old: rec {
-          patchPhase = ''
-            echo -java-home ${java} >> conf/sbtopts
-          '';
-        }
-      );
-    };
-  };
 
   nixpkgs = fetchTarball {
-    name   = "nixos-unstable-2021-10-27";
-    url    = "https://github.com/NixOS/nixpkgs/archive/a4bf44345706.tar.gz";
+    name = "nixos-unstable-2021-10-27";
```

Comment:
oh no man, you broke my perfect vertical alignment! 😱

Reply:
I'm using nixpkgs-fmt and so should you 😅

Reply:
I will when it supports vertical alignment 😬